binding.c

```diff
@@ -192,7 +195,7 @@ on_udx_stream_read (udx_stream_t *stream, ssize_t read_len, const uv_buf_t *buf)
   if (n->mode == UDX_NAPI_FRAMED) {
     if (n->frame_len <= read) {
       n->frame_len = -1;
-    } else if (n->read_buf_free == 0) {
+    } else if (n->read_buf_free < 2 * UDX_DEFAULT_MTU) {
```

Comment:
should we move 2 * mtu here to a defined contanst? UDX_MIN_BUF or something?

Reply:
Let's!